docs/user_manual/components.md

```diff
@@ -42,17 +42,17 @@ Uniform component updates are ones that update the same number of component of t
 
 | name      | data type | unit     | description             | required |  update  | valid values |
 | --------- | --------- | -------- | ----------------------- | :------: | :------: | :----------: |
-| `u_rated` | `double`  | volt (V) | rated line-line voltage | &#10004; | &#10060; |    `> 0`     |
+| `u_rated` | `double`  | Volt (V) | rated line-line voltage | &#10004; | &#10060; |    `> 0`     |
```

Comment:
Here and all the capitalization below.

Units in full name should be in lowercase, even if the units are derived from people's name.

https://www.nist.gov/pml/owm/writing-si-metric-system-units#:~:text=Unit%20Names%3A%20The%20names%20of,modifier%20%22Celsius%22%20is%20capitalized.

Capitalization of units for people's name is for abbreviation, not full name.

Reply:
i hate differences between languages.